uk.co.compucorp.civicrm.hrsampledata/tests/phpunit/CRM/HRSampleData/Cleaner/OptionValueTest.php

```diff
@@ -20,67 +20,82 @@ public function setUp() {
   }
 
   public function testProcessWithOptionGroupName() {
-    $optionValue = OptionValueFabricator::fabricate(['option_group_id' => $this->testOptionGroup['name']]);
-
-    $testOptionValue = $this->apiGet(
-      'OptionValue',
-      ['name' => $optionValue['name'], 'option_group_id' => $this->testOptionGroup['name']]
-    );
-
-    $this->assertEquals($optionValue['name'], $testOptionValue['name']);
+    $testOptionValue = OptionValueFabricator::fabricate(['option_group_id' => $this->testOptionGroup['name']]);
 
     $this->rows[] = [
       'name',
       $this->testOptionGroup['name'],
-      $optionValue['name'],
-      $optionValue['name'],
-      $optionValue['name'],
+      $testOptionValue['name'],
+      $testOptionValue['name'],
+      $testOptionValue['name'],
       '',
       '',
       0,
       0,
       0,
       '',
+      1,
     ];
 
     $this->runProcessor('CRM_HRSampleData_Cleaner_OptionValue', $this->rows);
 
     $optionValue = $this->apiGet(
       'OptionValue',
-      ['name' => $optionValue['name'], 'option_group_id' => $this->testOptionGroup['name']]
+      ['name' => $testOptionValue['name'], 'option_group_id' => $this->testOptionGroup['name']]
     );
     $this->assertEmpty($optionValue);
   }
 
-  public function testProcessWithOptionGroupTitle() {
-    $optionValue = OptionValueFabricator::fabricate(['option_group_id' => $this->testOptionGroup['name']]);
+  public function testProcessWithDeleteOnUninstallOff() {
+    $testOptionValue = OptionValueFabricator::fabricate(['option_group_id' => $this->testOptionGroup['name']]);
 
-    $testOptionValue = $this->apiGet(
+    $this->rows[] = [
+      'name',
+      $this->testOptionGroup['name'],
+      $testOptionValue['name'],
+      $testOptionValue['name'],
+      $testOptionValue['name'],
+      '',
+      '',
+      0,
+      0,
+      0,
+      '',
+      0,
+    ];
+
+    $this->runProcessor('CRM_HRSampleData_Cleaner_OptionValue', $this->rows);
+
+    $optionValue = $this->apiGet(
       'OptionValue',
-      ['name' => $optionValue['name'], 'option_group_id' => $this->testOptionGroup['name']]
+      ['name' => $testOptionValue['name'], 'option_group_id' => $this->testOptionGroup['name']]
     );
+    $this->assertEquals($testOptionValue['name'], $optionValue['name']);
+  }
 
-    $this->assertEquals($optionValue['name'], $testOptionValue['name']);
+  public function testProcessWithOptionGroupTitle() {
```

Comment:
The test title should probably be changed to reflect that delete_on_uninstall is on.